src/Compilers/VisualBasic/Portable/Syntax/Syntax.xml

```diff
@@ -5037,6 +5037,77 @@ Unify Enum declarations with other block type declarations.
     </node-structure>
 
     <!--****************
+      -  TupleExpression
+      ******************-->
+    <node-structure name="TupleExpressionSyntax" parent="ExpressionSyntax">
+      <description>Represents a tuple literal expression.</description>
+      <spec-section>42.42.42</spec-section>
+      <grammar>TupleExpression</grammar>
+
+      <node-kind name="TupleExpression">
+      </node-kind>
+
+      <child name="OpenParenToken" kind="OpenParenToken">
+        <description>The "(" token</description>
+      </child>
+
+      <child name="Arguments" optional="false" list="true" optional-elements="false" separator-kind="CommaToken" separator-name="CommaTokens" kind="@SimpleArgumentSyntax">
+        <description>The list of tuple arguments.</description>
+      </child>
+      
+      <child name="CloseParenToken" kind="CloseParenToken">
+        <description>The ")" token</description>
+      </child>
+    </node-structure>
+
+      <!--****************
+      -  TupleType
+      ******************-->
+    <node-structure name="TupleTypeSyntax" parent="TypeSyntax">
+      <description>Represents a tuple type expression.</description>
+      <spec-section>42.42.42</spec-section>
+      <grammar>TupleType</grammar>
+
+      <node-kind name="TupleType">
+      </node-kind>
+
+      <child name="OpenParenToken" kind="OpenParenToken">
+        <description>The "(" token</description>
+      </child>
+
+      <child name="TupleElements" optional="false" list="true" optional-elements="false" separator-kind="CommaToken" separator-name="CommaTokens" kind="@TupleElementSyntax">
+        <description>The list of tuple arguments.</description>
+      </child>
+      
+      <child name="CloseParenToken" kind="CloseParenToken">
+        <description>The ")" token</description>
+      </child>
+    </node-structure>
+
+    <!--********************
+      -  TupleElementSyntax
+      **********************-->
+    <node-structure name="TupleElementSyntax" parent="VisualBasicSyntaxNode">
+      <description>Represents a single declaration of a tuple element. </description>
+      <spec-section>42.42.42</spec-section>
+      <grammar>TupleElement</grammar>
+
+      <node-kind name="TupleElement"></node-kind>
+
+      <child name="IdentifierName" optional="true" kind="IdentifierName">
+        <description>An identifier naming a tuple element.</description>
+      </child>
+
+      <child name="AsKeyword" optional="true" kind="AsKeyword">
+        <description>The "As" keyword.</description>
```

Comment:
This should probably be an AsClauseSyntax, which is the way similar situations are handled elsewhere in the syntax trees.